app-backend/project/Dependencies.scala

```diff
@@ -38,7 +38,7 @@ object Version {
   val dropbox = "3.0.9"
   val elasticacheClient = "1.1.1"
   val ficus = "1.4.0"
-  val flyway = "5.2.4"
+  val flyway = "6.0.8"
```

Comment:
This is not the most up-to-date version of Flyway. I just updated to the most recent version with PostgreSQL 12.x support.